dtmi/tinkergen/wioterminal_smellsensor-1.json

```diff
@@ -0,0 +1,48 @@
+{
+    "@id": "dtmi:tinkergen:wioterminal_smellsensor;1",
+    "@type": "Interface",
+    "@context": "dtmi:dtdl:context;2",
+    "displayName": "wioterminal_smellsensor",
```

Comment:
Consider using Uppercase and white spaces

Reply:
Hi @rido-min  

This model is related to the Wioterminal, actually seedk and tinkergen from the same company which is seeed studio.

Do you mean using the Uppercase and white spaces on the displayName, 
If uppercase and white spaces are optional, can I just keep this content then you merge it?

Reply:
My suggestion was about the `displayName`, instead of `wioterminal_smellsensor` change it to `WioTerminal Smell Sensor`.

Once you update the PR info (below) I will merge

![image](https://user-images.githubusercontent.com/14916339/133137894-70a05296-2847-4303-94ce-1177aeb31d8f.png)

Reply:
Hi rido-min
I have changed it please check it again. it should be ok.

Reply:
Thanks, changes LGTM. However, as this is the first submission under the `tinkergen` namespace we still need to collect some information about your company, device information etc. 

You can edit the first message in the PR thread.

Reply:
Hi rido-min
I added the information on the first message, please check it.